runtime/kubernetes/container.go

```diff
@@ -378,6 +370,10 @@ func (c *client) WaitContainer(ctx context.Context, ctn *pipeline.Container) err
 				continue
 			}
 
+			// cst.State has details about the cst.Image's exit.
+			// cst.LastTerminationState has details about the kubernetes/pause image's exit.
+			// cst.RestartCount is 1 at exit due to switch from kubernetes/pause to final image.
```

Comment:
Should these be removed?

Reply:
I don't think removing it is necessary since it has a purpose 👍 

More so, I wanted to double check that this was copied in during developmment and then left unintentionally